ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -223,7 +223,7 @@ def euclidean(cls, pos1, pos2):
     def distance(cls, waypoints, wp1, wp2):
         dl = 0
         n = len(waypoints)
-        for i in range(wp1+1, wp2):
+        for i in range(wp1 + 1, wp2 + 1):
```

Comment:
I find this a bit convoluted. Would it work the same doing this?

    for i in range(wp1, wp2):
        dl += cls.euclidean(waypoints[i % n], waypoints[(i+1)%n])
?